src/modules/private.ts

```diff
@@ -464,6 +464,7 @@ export default class Private {
    */
   async createOrder(
     params: PartialBy<ApiOrder, 'clientId' | 'signature'>,
+    genericParams: GenericParams = {},
```

Comment:
Can you put this param after `positionId`? Otherwise it will be a breaking change when folks upgrade

Reply:
oh right - very good catch!

updating